woocommerce/woocommerce-android
# Coupon validation

Closes: #9065

**⚠️ Should be reviewed and merged after the [core PR](https://github.com/woocommerce/woocommerce-android/pull/9024) is merged.**

### Description

This PR adds validation of the coupon codes added to an order. Validation doesn't allow adding a coupon to an order if the coupon is not found in the store.

### Testing instructions

Try to enter an invalid coupon. Observe that it's not possible to add it to an order.
